xacro/solo.urdf.xacro

```diff
@@ -8,7 +8,7 @@
   <xacro:property name="color_name" value="grey" />
   <xacro:property name="color" value="0.8 0.8 0.8" />
   <xacro:property name="opacity" value="1.0" />
-  <xacro:property name="mesh_ext" value="obj" />
+  <xacro:property name="mesh_ext" value="stl" />
```

Comment:
Out of interest: What's the motivation to change from obj to stl? Why wasn't stl used in the first place?